pages/index.tsx

```diff
@@ -19,13 +19,13 @@ export default function Mastermind() {
   return (
     <>
       <NextSeo
-        title="zkMastermind"
-        description="An on-chain code-breaker game built with zero-knowledge proofs"
+        title="zkMastermind x zkVerify"
+        description="An on-chain code-breaker game built with zero-knowledge proofs with verification by zkVerify"
         openGraph={{
-          url: "https://zk-mastermind.vercel.app",
-          title: "zkMastermind",
+          url: "https://zkverify.io",
```

Comment:
`url: "https://zkmastermind.zkverify.io"`
This should be the canonical url for the page and used for SEO & social media sharing

Reply:
Done